src/test/kotlin/uk/gov/dluhc/printapi/rest/DeprecatedGetCertificateSummaryByApplicationIdIntegrationTest.kt

```diff
@@ -0,0 +1,140 @@
+package uk.gov.dluhc.printapi.rest
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.http.MediaType
+import uk.gov.dluhc.printapi.config.IntegrationTest
+import uk.gov.dluhc.printapi.database.entity.PrintRequestStatus.Status
+import uk.gov.dluhc.printapi.database.entity.SourceType
+import uk.gov.dluhc.printapi.models.CertificateSummaryResponse
+import uk.gov.dluhc.printapi.models.PrintRequestStatus
+import uk.gov.dluhc.printapi.models.PrintRequestSummary
+import uk.gov.dluhc.printapi.testsupport.bearerToken
+import uk.gov.dluhc.printapi.testsupport.testdata.anotherValidEroId
+import uk.gov.dluhc.printapi.testsupport.testdata.entity.buildCertificate
+import uk.gov.dluhc.printapi.testsupport.testdata.entity.buildPrintRequest
+import uk.gov.dluhc.printapi.testsupport.testdata.entity.buildPrintRequestStatus
+import uk.gov.dluhc.printapi.testsupport.testdata.getBearerToken
+import uk.gov.dluhc.printapi.testsupport.testdata.model.buildElectoralRegistrationOfficeResponse
+import java.time.Instant
+import java.time.ZoneOffset
+import java.time.temporal.ChronoUnit.SECONDS
+
+@Deprecated(
+    """
+        The endpoint /eros/{ERO_ID}/certificates/applications/{APPLICATION_ID} has been deprecated 
+        and this test class should be removed when the implementation is removed.    
+    """
+)
+internal class DeprecatedGetCertificateSummaryByApplicationIdIntegrationTest : IntegrationTest() {
```

Comment:
This class is actually just a rename of `GetCertificateSummaryByApplicationIdIntegrationTest` with the `@Deprecated` annotation added